inst/extremes/extremes.Rmd

```diff
@@ -1,27 +1,27 @@
-
-![](http://www.usgs.gov/newsroom/images/2015-04-07/usgs_logo.png)
-
-## Extremes Report  
-
-**Location:**  `r getValue(ts,'locationNumber',required=TRUE)` `r getValue(ts,'locationName',required=TRUE)`
-   
-**Period** `r getInput(ts, 'startDate')` to `r getInput(ts, 'endDate')`
-
-**Time Series:** `r getValue(ts,'inputDataDescriptions1')`, `r getValue(ts,'inputDataDescriptions2')`, `r getValue(ts,'inputDataDescriptions3')`
-
-`r if (length(author) != 0) { paste("**Author:**",author[[1]]) }`
-
-`r if (length(author) == 0) { paste("**Author:** Username")}`
-
-**Created:**  `r Sys.time()`
-
-***
-   
-Maximum for period, based on 365 equivalent days of record. 
-   
-```{r,echo=FALSE,result='asis',comment=NA}
-tbl <- extremesTable(ts)
-# formTable <- padTable(tbl)
-# cat(formTable)
-kable(tbl)
-```
+
+![](http://www.usgs.gov/newsroom/images/2015-04-07/usgs_logo.png)
+
+## Extremes Report  
+
+**Location:**  `r getReportMetadata(ts,'locationNumber',required=TRUE)` `r getReportMetadata(ts,'locationName',required=TRUE)`

```

Comment:
Yeah, I noticed the same too. It was a remnant of multiple people doing it simultaneously for 2 reports I believe. Consolidation/unification of both is definitely the goal.